chapters/lang/introduction.tex

```diff
@@ -84,6 +84,8 @@ \section{Values, Types, Terms and Results}
 
 Evaluating expressions may either succeed or fail. A tagged union of successfully evaluated result or failure with information describing what failed is called a \emph{result}\index{result}. Terms and results are the basic blocks of representing information in Intentio.
 
+There is also a special subgroup of results named  \emph{unit}\index{unit}. It is characterized by content of term. Tho both \emph{value} and \emph{type} are set to word \emph{unit}.
```

Comment:
Unit is not a special case, it's just a predefined term :) Please, remove this sentence